packages/cadl-vs/src/Exceptions.cs

```diff
@@ -27,10 +27,10 @@ public CadlServerNotFoundException(string fileName, Exception? innerException =
             : base(string.Join("\n", new string[]
             {
                 $"Cadl server executable was not found: '{fileName}' is not found. Make sure either:",
-                " - cadl is installed locally at the root of this workspace.",
+                " - cadl is installed locally at the root of this workspace or in a parent directory.",
                 " - cadl is installed globally with `npm install -g @cadl-lang/compiler'.",
                 " - cadl server path is configured with https://github.com/microsoft/cadl/blob/main/packages/cadl-vs/README.md#configure-cadl-visual-studio-extension."
-            }, innerException))
+            }), innerException)
```

Comment:
oops

Reply:
yeah realized it was displaying `string[]` and then some exception that should have been caught. kinda intresting that string.Join accept exceptions as an input